modules/xfem/include/userobjects/GeometricCutUserObject.h

```diff
@@ -182,6 +182,18 @@ class GeometricCutUserObject : public CrackFrontPointsProvider
    */
   bool shouldHealMesh() const { return _heal_always; };
 
+  /**
+   * Get a consistent boolean telling which side the node belongs to relative to the cut
```

Comment:
What do you mean by a 'consistent boolean'? I would suggest maybe renaming this class (or explaining what the consistent boolean is).

Reply:
I want the function to return a boolean that is persistent across the entire simulation. The boolean itself should not contain any realistic information though. Can you teach me the English here, should it "consistent" or "persistent"?